build.sh

```diff
@@ -73,7 +73,7 @@ fi
 mkdir -p $WORKSPACE_DIR/build
 cd $WORKSPACE_DIR/build
 
-cmake .. -DCMAKE_BUILD_TYPE=$BUILD_TYPE && make -j$THREADS
+cmake .. -B./ -H../ -DCMAKE_BUILD_TYPE=$BUILD_TYPE && make -j$THREADS
```

Comment:
I stand corrected then. If it is causing issues then by all means, keep it (and we should probably add those arguments to the main computer-vision repo, though I do wonder whether those undocumented arguments will still exist down the road). I would be interested to know why this would be an issue on your system though.